### ISSUE TITLE
Submodule commit for ethereum/tests only exists in a fork

### ISSUE THREAD

**Issue by Rizato** [openethereum/openethereum#11668]
- **OpenEthereum version**: None
- **Operating system**: Linux
- **Installation**: cargo
- **Fully synchronized**: no 
- **Network**: none
- **Restarted**: yes

**Steps to reproduce**
1. Add `common-types = { git = "https://github.com/openethereum/openethereum" }` to `Cargo.toml` of another project. 
1. Run `cargo build`

**Expected Behavior**

Completed cargo build

**Actual behavior**

```
error: failed to get `common-types` as a dependency of package `[package] (/path/to/project)`

Caused by:
  failed to load source for dependency `common-types`

Caused by:
  Unable to update https://github.com/openethereum/openethereum

Caused by:
  failed to update submodule `ethcore/res/ethereum/tests`

Caused by:
  object not found - no match for id (06acfb48aee71ecb57a9ca991cf0f57b630e3469); class=Odb (9); code=NotFound (-3)
```

While searching down the issue I came across the same issue from 16 days ago. 
https://github.com/openethereum/openethereum/issues/11631 that offers a workaround, but not a solution. 

It appears that the commit is searchable on github, but does not exist in the repository anymore. 
https://github.com/ethereum/tests/commits/06acfb48aee71ecb57a9ca991cf0f57b630e3469

Taking a look at the network graph for `ethereum/tests` we can see that commit is only in a fork of the project, not the main project. 

I think the resolution here is to use a commit that exists in the root ethereum/tests repo.

**Comment by ordian** [comment]
@adria0 do you think this will be fixed by #11597?

**Comment by Rizato** [comment]
Yes, `5841af` is in the master branch.

**Comment by adria0** [comment]
> @adria0 do you think this will be fixed by #11597?

It should. I'm just finishing checking the main problems with the current development branch. I open the PR asap to review.
